client/consensus/manual-seal/src/consensus/babe.rs

```diff
@@ -190,8 +198,10 @@ where
 		+ UsageProvider<B>
 		+ ProvideRuntimeApi<B>,
 	C::Api: BabeApi<B>,
+	P: Send + Sync + 'static,
```

Comment:
```suggestion
	P: Send + Sync,
```
Lifetime bound not strictly required here